eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -162,9 +162,15 @@ jobs:
           ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
         displayName: 'Verify Readmes'
 
+      - pwsh: |
+            $diffs = git diff origin/master HEAD --name-only -- *.md
+            echo "##vso[task.setvariable variable=diffMarkdown]$diffs"
+        displayName: 'Find diff files'
+
       - template: ../../../common/pipelines/templates/steps/verify-links.yml
         parameters:
           Directory: sdk/${{ parameters.ServiceDirectory }}
```

Comment:
Your diff command will find all files in the PR not just files under the service directory so we may need to figure out what Directory we want to pass here.

Reply:
I have tried this but not working. I did not check whether it contains .git directory in working directory. I can test more if we want to approach with this way.

Reply:
The directory is used for `$workingDirectory` in Verify-Link.yml. It is not for markdown files anymore. We don't want to limit the diff to check only under the directory, which might miss the root readme.